finance/proposal-calls/cycle3.md

```diff
@@ -0,0 +1,90 @@
+
+
+# Funding goals
+
+Funding requests (FR) shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, and anything else broadly Project related.
+
+FRs shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+FRs need to clearly identify how the the funding will be used (e.g. "pay $YYY (US) to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+
+# Funding amount
+
+We anticipate a maximum of **$400k** (US) to be available for this funding cycle. This is significantly larger than previous cycles. This means this cycle may be limited more by the available time of current community members rather than by the amount of funding available.  We therefore urge the community to **aim high**, either by ambitious and/or expensive proposals for their own work, or by bringing in new members via the “open hire” process that is new in this cycle (outlined below).
+
+
+# What can the funding be used for?
+
+Money can be used for travel, subscriptions, etc, in addition to direct financial support to  individuals.
+
+The funds come from two grants: a [Moore Foundation Data-Driven Discovery Grant](https://www.moore.org/grant-detail?grantId=GBMF8435) and a NASA ROSES grant.
+
+
+
+* Both grants include funds for basic maintenance and development on all aspects of the Project. 
+* The Moore grant has a focus on mentoring and spectroscopy, 
+* The NASA grant has a focus on support for project-wide infrastructure, affiliated packages, and Learn.
+
+The Finance Committee assumes responsibility for ensuring that all awards made are consistent with any applicable funding agency rules.
+
+
+# Funding Request process
+
+Funding Requests (FRs) should be posted as pull requests to github.com/astropy/astropy-project/ with the initial title text "Phase 3 Funding:". The goal is to make this process as easy as possible for everyone, so we impose a limit of 100 lines assuming 80 char/line. We provide a template below. 
+
+We envision a two stage FR process to enable the community to comment and improve ideas, teams with similar ideas can merge to avoid duplicated work, etc.
+
+
+
+* July 1: **_Draft_** FR Deadline (after this date, no new requests will be accepted)
+* Six week iteration and discussion period during which draft FRs can be fleshed out and modified
+* Aug 12: Finalized FRs are due
+* Finance Committee reviews for compatibility with legal conditions
+* Aug 22: voting begins.
+
```

Comment:
The Drafts that are due on July 1 can be very minimal. Our intention is get all the rough ideas out on the table as soon as possible. Since this funding opportunity was discussed at the Coordination Meeting, we think folks have been given enough time to formulate their ideas.